%YAML:1.0
---
Robot:
   baseLinkPosCoord: [ 0., 0. ]
Chassis:
   chassisfileName: "ramaxx.png"
   chassisImageCenter: [ -1., -1. ]
   chassisImageValueOffset: 0.
   chassisImageValueScale: 1.
   chassisModelYSize: 0.417
   chassisPosRobot: [ 0., 0. ]
   testChassis: 0
Wheels:
   wheelPosRobotX: 0.23
   wheelPosRobotFrontY: 0.2
   wheelPosRobotRearY: 0.2
   wheelJointPosFront: [ 0., 0. ]
   wheelRadiusFront: 0.09652
   wheelWidthFront: 0.14
   wheelLatRadiusFront: 0.0
   wheelJointPosRear: [ 0., 0. ]
   wheelRadiusRear: 0.09652
   wheelWidthRear: 0.14
   wheelLatRadiusRear: 0.0
   frontWheelsTurnable: 1
   wheelRotTestSteps: 0
   wheelRotTestStepSize: 0   
Proc:
   numAngleStep: 360
   heightScale: 1000.0
   mapBaseHeight: 10000
   wheelGroundLevel: 20000
   maxHeight: 30000
   pixelSize: 0.00625
   imagePosBLMinX: 0.75
   imagePosBLMinY: -1.0
   validThresholdFactor: 0.95
   convertImage: 0


